tool/build/pledge.c

```diff
@@ -555,10 +555,16 @@ void ApplyFilesystemPolicy(unsigned long ipromises) {
       if ((p = getenv("TMPDIR"))) {
         UnveilIfExists(
             __join_paths(buf, sizeof(buf), p, ".ape-" APE_VERSION_STR), "rx");
+        UnveilIfExists(__join_paths(buf, sizeof(buf), p,
```

Comment:
Probably these could be wrapped in a condition as well to only run on xnu, but I don't know what / if there is an appropriate check available in this context. Same for bellow.